Mail/Views/Unavailable Mailbox/LockedMailboxView.swift

```diff
@@ -16,44 +16,64 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import InfomaniakDI
 import MailCore
 import MailCoreUI
 import MailResources
 import SwiftUI
 
 struct LockedMailboxView: View {
+    @LazyInjectService private var accountManager: AccountManager
+
     @Environment(\.dismiss) private var dismiss
 
+    let mailbox: Mailbox
```

Comment:
As we only need the email, maybe the view can only take the email instead of the mailbox?